pkgs/tools/security/nitrokey-app2/default.nix

```diff
@@ -0,0 +1,84 @@
+{ lib
+, fetchFromGitHub
+, fetchpatch
+, buildPythonApplication
+, callPackage
+, libsForQt5
+, pynitrokey
+, pythonOlder
+, pyudev
+}:
+
+let
+  pyqt5-stubs = callPackage ./pyqt5-stubs.nix { };
```

Comment:
This sounds more like something top-level which should be under pythonPackages

Reply:
As discussed following your hint in https://github.com/NixOS/nixpkgs/pull/230545#discussion_r1192837426 this would lead to collisions because it requires a different pyqt5 (with many optional features enabled) and would collide with the "normal" pyqt5 in pythonPackages.